cuda/test/base/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 ginkgo_create_cuda_test(array)
 ginkgo_create_cuda_test(cuda_executor)
-ginkgo_create_test(index_set)
+ginkgo_create_cuda_test(index_set)
 if(GINKGO_HAVE_HWLOC)
     find_package(NUMA REQUIRED)
     ginkgo_create_cuda_test(cuda_executor_topology ADDITIONAL_LIBRARIES NUMA::NUMA)
-endif()
+endif ()
 ginkgo_create_cuda_test(exception_helpers)
 ginkgo_create_cuda_test(kernel_launch)
 ginkgo_create_cuda_test(lin_op)
 ginkgo_create_cuda_test(math)
-ginkgo_create_test(memory)
-ginkgo_create_cuda_test(scoped_device_id)
+ginkgo_create_cuda_test(memory)
```

Comment:
I do not like using unnecessary nvcc compilers to compile the no-cuda code. We usually use the host compiler to call ginkgo not the nvcc compiler

Reply:
This can be changed easily, I tend to agree

Reply:
do you mean it will be changed in another pr?

Reply:
It requires a few more changes, but by removing the `ref` resource type and letting the `-j` parameter take care of overall test parallelism, I can do it ;)

Reply:
done